content/2.0.html.haml

```diff
@@ -167,6 +194,11 @@ title: "Jenkins 2.0"
   We're very interested in your feedback on what you think of the Jenkins 2.0
   preview releases.
 
+%p
+  To experience Jenkins 2.0 properly, we recommend a different Jenkins home directory.
```

Comment:
Without linking to documentation explaining what this means, I'm not sure this is a helpful suggestion.